libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/lifecycle/UICoordinator.kt

```diff
@@ -39,7 +47,9 @@ abstract class UICoordinator<T : ViewGroup>(
             }
         }.invokeOnCompletion {
             attachedObserver?.onDetached(mapboxNavigation)
-            viewGroup.removeAllViews()
+            if (removeViewsOnDetached) {
```

Comment:
We're discussing if we even need to `removeAllViews`

The idea is the Binder will always add view on `bind`, but they do not `unbind`. The `removeAllViews` here guarantees that a coordinator will not leave a stale view behind.

This gives us the ability to remove UICoordinators if we want. But, it is arguable that we will not need this. 100% of our usages assume the top level view group is